terraform/service_appsync.tf

```diff
@@ -0,0 +1,215 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+resource "google_service_account" "appsync" {
+  project      = var.project
+  account_id   = "en-verification-appsync-sa"
```

Comment:
Limit is 20 chars

Reply:
https://github.com/google/exposure-notifications-verification-server/blob/1451864d0a3a4a111a38367b0f8eb0a7bc87a785/terraform/service_cleanup.tf#L17

are these existing values okay?